lib/vsc/install/shared_setup.py

```diff
@@ -268,7 +268,20 @@ def build_setup_cfg_for_bdist_rpm(target):
 def action_target(target, setupfn=setup, extra_sdist=[]):
     # EXTRA_SDIST_FILES.extend(extra_sdist)
 
-    cleanup()
+    do_cleanup = True
+    try:
+        # very primitive check for install --skip-build
+        # in that case, we don't mind "leftover build";
+        # it's probably intentional
+        install_ind=sys.argv.index('install')
+        build_skip=sys.argv.index('--skip-build')
+        if build_skip > install_ind:
```

Comment:
as the comment above says, it's a very primitive check to see if the `--skip-build` is an `install` option